include/tables/object_table.h

```diff
@@ -0,0 +1,411 @@
+/**
+ * Object Table
+ *
+ * DEFINE_OBJECT should be used for normal objects
+ *    - Argument 1: Name of the object segment in spec
+ *    - Argument 2: Enum value for this object
+ *
+ * DEFINE_OBJECT_UNSET and DEFINE_OBJECT_NULL are needed to define empty entries from the original game
+ */
+/* 0x0000 */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_0)
```

Comment:
I have a vague memory that object 0 cannot be used but I don't remember how I would know that

I looked at the code and this may back it up:

https://github.com/zeldaret/oot/blob/7551dc2b71f7c6587f96c498cbd62cfd05aa292c/src/code/z_scene.c#L86

another hint may be https://github.com/zeldaret/oot/blob/7551dc2b71f7c6587f96c498cbd62cfd05aa292c/src/code/z_scene.c#L279

I can't really get a definitive opinion on if object id 0 is usable or not from code (but I kinda remember not being able to use it for some reason), if it indeed isn't it would be nice to document it here

Reply:
I think you're right that object id 0 is no good because the system uses the negative value of the id to mark an object as not being loaded yet, so I 0 could easily cause issues even if it somehow worked

Reply:
I can add a comment about this in the table, but maybe we should change the enum value for 0 to `OBJECT_NONE` or something and potentially use that in the code where there is `id = 0`?